handlers/handler.go

```diff
@@ -161,7 +163,7 @@ func (handler *SQSHandler) HandleSQSMessage(m *mq.Message) error {
 	}
 
 	// remove completed jobs
-	RemoveCompletedJobs(jobNameList)
+	//RemoveCompletedJobs(jobNameList)
```

Comment:
Is this intentional?

Reply:
Fixed in 58015ab91c638aad9adb29fb6e60cbc947327845
Thanks!